q2_qemistree/_itol_metadata.py

```diff
@@ -11,15 +11,16 @@
 import pandas as pd
 import seaborn as sns
 import click
-from qiime2 import Artifact
+import biom
+from qiime2 import Artifact, Metadata
 
 
 def classyfire_to_colors(classified_feature_data: pd.DataFrame,
-                         feature_data_column: str, color_palette: str):
+                         classyfire_level: str, color_palette: str):
```

Comment:
Can we use `feature_data_column` instead of `classyfire_level`

Reply:
@anupriyatripathi done!